### ISSUE TITLE
Inline Styling Issue

### ISSUE THREAD

**Issue by francis450** [baqar-abbas/Trial-Project-Review#3]
The overall code in the both html and css files displays as required but the presenter has been reminded to try and use html and css best practices which prohibits the use of inline styling.
Happy Coding :)

**Comment by baqar-abbas** [comment]
Yes Francis as we discussed, in all my future projects I am not using inline css and this was trial project so I will remove all inline styling and put it in my css file.
Thanks for your valuable peer to peer review.
Happy Coding :)
